### ISSUE TITLE
Update Image Carousel title header level 

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/cgov-digital-platform#4043]
The image carousel title, when present, is currently an `<h4>`.  During an a11y audit, it was found that this should be changed to  not be a header at all but be styled like a header instead.

> **ESTIMATE** 1

### What's the expected change?
- removal of `<h4>` heading and replacement of styling

### Developer notes
- `block-content--cgov-image-carousel--full.html`
